plugins/catalog-backend/api-report.md

```diff
@@ -601,6 +613,7 @@ export const processingResult: Readonly<{
     newEntity: Entity,
   ) => CatalogProcessorResult;
   readonly relation: (spec: EntityRelationSpec) => CatalogProcessorResult;
+  readonly refresh: (entityRef: string, key: string) => CatalogProcessorResult;
```

Comment:
As far as we can tell at this point we only need to refresh emit refresh keys for the entity that is currently being processed. I suggest we drop the entityRef from the result and leave it to be added by the `updateProcessedEntity` method. In that case the `setRefreshKeys` method is no longer needed on the database.

Reply:
Yea, makes sense and much nicer imo!